robottelo/config/validators.py

```diff
@@ -23,7 +23,9 @@
         ),
         Validator('server.admin_password', default='changeme'),
         Validator('server.admin_username', default='admin'),
-        Validator('server.deploy_workflow', must_exist=True),
+        Validator('server.deploy_workflow', must_exist=True, is_type_of=dict),
```

Comment:
i am playing with an idea, that accepting the old format could be accepted too (for backward compatibility). in such scenario, the value would be automatically treated as `PRODUCT` workflow.

Reply:
Just a note: we can use Dynaconf hooks for that. https://www.dynaconf.com/advanced/?h=hook#hooks-for-the-solution

Reply:
@rplevka I have introduced config migration. Please see the latest changes.